wandb/sdk/lib/disabled.py

```diff
@@ -11,3 +16,14 @@ def __getitem__(self, key):
             val = SummaryDisabled(val)
             self[key] = val
         return val
+
+
+class RunDisabled:
+    """Compatibility class for integrations that explicitly check for wandb.RunDisabled."""
+
+    def __getattribute__(self, name: str) -> Any:  # type: ignore
```

Comment:
`__getattr__` should be the default as it's safer; `__getattribute__` can easily result in infinite recursion though it works in this case

Reply:
nice catch, thanks. ide autocomplete, i trust it too much :(